src/target/stm32l4.c

```diff
@@ -187,6 +191,9 @@ typedef enum stm32l4_family {
 	STM32L4_FAMILY_WLxx,
 } stm32l4_family_e;
 
+/* recommend augmenting this structure with flash block size,
+	which would get rid of the hard-coded magic numbers in
+	stm32l4_attach, but I have not implemented that yet */
```

Comment:
As this is a multi-line comment, please use the multi-line comment style for the code base:

```suggestion
/*
 * XXX: Recommend augmenting this structure with flash block size,
 * which would get rid of the hard-coded magic numbers in
 * stm32l4_attach, but that has not been implemented yet
 */
```